manifests/send_hit.pp

```diff
@@ -20,16 +20,14 @@
 #   A preceeding comment to add to the configuration file.
 define squid::send_hit (
   Enum['allow', 'deny']
-          $action  = 'allow',
+  $action  = 'allow',
```

Comment:
This is also really one line. It's repeated in a few places, basically every define.

Reply:
fixed in all files